app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
```

Comment:
- [ ] Kindly ensure the signup page matches the design given, check the comparison below:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/211101564-bbde26f2-0f38-4a74-91cf-c2ab763a2762.png) |  ![image](https://user-images.githubusercontent.com/61408860/211101616-7bbb8cfc-7183-4877-88cc-c143e5b86ab5.png)